src/test/scripts/helm_install.sh

```diff
@@ -3,6 +3,18 @@
 set -e
 set -x
 
+getClusterType() {
+  local currentContext=$1
+
+  case "${currentContext}" in
+    *eks*) echo EKS;;
+    *aks*) echo AKS;;
+    *gke*) echo GKE;;
+    *shared-dev*|*default-context*) echo KITT;;
+    *) echo CUSTOM;;
+  esac
+}
+
```

Comment:
Why move this? Isn't the bash version check logic here because of the use of this syntax?

Reply:
The version check will be executed before the function is parsed anyway, regardless of location.